docs/content/users/usage/troubleshooting.md

```diff
@@ -8,6 +8,7 @@ Things might go wrong! In addition to this page, consider checking [Stack Overfl
 * Temporarily disable firewalls, VPNs, tunnels, network proxies, and virus checkers while you’re troubleshooting.
 * Temporarily disable any proxies you’ve established in Docker’s settings.
 * Use [`ddev debug dockercheck`](commands.md#debug-dockercheck) and [`ddev debug test`](commands.md#debug-test) to help sort out Docker problems.
+* Make sure you do not have disk space problems on your computer. This can be especially tricky on WSL2, where you need to check both the host disk space and WSL2 disk space as well.
```

Comment:
```suggestion
* Make sure you do not have disk space problems on your computer. This can be especially tricky on WSL2, where you need to check both the main Windows disk space and WSL2 disk space as well.
```